CHANGELOG.md

```diff
@@ -0,0 +1,6 @@
+# Release Notes
+
+## [v6.0.0 (unreleased)](https://github.com/laravel/framework/compare/v5.0.3...v6.0.0)
```

Comment:
This should be: 	## [v6.0.0 (unreleased)](https://github.com/laravel/scout/compare/v5.0.3...v6.0.0)